src/pages/Flow/Settings/Schedules.vue

```diff
@@ -517,6 +530,29 @@ export default {
                       </template>
                       This schedule overrides default parameters.
                     </v-tooltip>
+
```

Comment:
I like this warning but could we make it clearer at a glance that this schedule will not be used - maybe reduce the opacity of flow schedule cards to show they are disabled?